builders/pallets-precompiles/precompiles/proxy.md

```diff
@@ -0,0 +1,124 @@
+
+ - **addProxy**(*address* delegate, *ProxyType* proxyType, *uint32* delay) — registers a proxy account for the sender after a specified number of delay blocks (generally zero). Will fail if a proxy for the caller already exists
+ - **removeProxy**(*address* delegate, *ProxyType* proxyType, *uint32* delay) — removes a registered proxy for the sender
+ - **removeProxies**() — removes all of the proxy accounts delegated to the sender
+ - **isProxy**(*address* real, *address* delegate, *ProxyType* proxyType, *uint32* delay) — returns a boolean, true if the delegate address is a proxy of type proxyType, for address real, with the specified delay
+ 
+## Proxy Types {: #proxy-types }
+
+There are multiple types of proxy roles that can be delegated to accounts, where are represented in `Proxy.sol` through the `ProxyType` enum. The following list includes all of the possible proxies and the type of transactions they can make on behalf of the primary account:
+
+ - **Any** — the any proxy will allow the proxy account to make any type of transaction
+ - **NonTransfer** — the non-transfer proxy will allow the proxy account to make any type of transaction, except for balance transfers
+ - **Governance** - the governance proxy will allow the proxy account to make any type of governance related transaction (includes both democracy or council pallets)
+ - **Staking** - the staking proxy will allow the proxy account to make staking related transactions
+ - **CancelProxy** - the cancel proxy will allow the proxy account to reject and remove delayed proxy announcements (of the primary account)
+ - **Balances** - the balances proxy will allow the proxy account to only make balance transfers
+ - **AuthorMapping** - this type of proxy account is used by collators to migrate services from one server to another
+ - **IdentityJudgement** - the identity judgement proxy will allow the proxy account to judge and certify the personal information associated with accounts on Polkadot
+
+## Interact with the Solidity Interface {: #interact-with-the-solidity-interface }
+
+### Checking Prerequisites {: #checking-prerequisites } 
+
+The below example is demonstrated on Moonbase Alpha, however, similar steps can be taken for Moonbeam and Moonriver. You should:  
+
+ - Have MetaMask installed and [connected to Moonbase Alpha](/tokens/connect/metamask/){target=_blank}
+ - Have an account with some DEV tokens.
+ - Have a second account that you control to use as a proxy account (funding optional).
+ --8<-- 'text/faucet/faucet-list-item.md'
+
+### Remix Set Up {: #remix-set-up } 
+
+1. Get a copy of [`Proxy.sol`](https://github.com/PureStake/moonbeam/blob/master/precompiles/proxy/Proxy.sol){target=_blank}
+2. Copy and paste the file contents into a [Remix file](https://remix.ethereum.org/){target=_blank} named `Proxy.sol`
+
+![Copying and Pasting the Proxy Interface into Remix](/images/builders/pallets-precompiles/precompiles/proxy/proxy-1.png)
```

Comment:
I think this needs to be updated because the numbers in the image don't align with the numbered steps. I think we should do something like:

To get started, get a copy of [`Proxy.sol`](https://github.com/PureStake/moonbeam/blob/master/precompiles/proxy/Proxy.sol){target=_blank} and take the following steps:

1. Click on the **File explorer** tab
2. Copy and paste the file contents into a [Remix file](https://remix.ethereum.org/){target=_blank} named `Proxy.sol`

Reply:
I'll create a ticket to update all of the other precompile pages because they all do this 😅